common/eth2_network_config/built_in_network_configs/chiado/config.yaml

```diff
@@ -150,7 +43,95 @@ BELLATRIX_FORK_EPOCH: 180 # Mon Oct 10 2022 14:00:00 GMT+0000
 # Capella
 CAPELLA_FORK_VERSION: 0x0300006f
 CAPELLA_FORK_EPOCH: 244224 # Wed May 24 2023 13:12:00 GMT+0000
+# Deneb
+DENEB_FORK_VERSION: 0x0400006f
+DENEB_FORK_EPOCH: 516608 # Wed Jan 31 2024 18:15:40 GMT+0000
+
+
+# Time parameters
+# ---------------------------------------------------------------
+# 5 seconds
+SECONDS_PER_SLOT: 5
+# 6 (estimate from xDai mainnet)
+SECONDS_PER_ETH1_BLOCK: 6
+# 2**8 (= 256) epochs ~5.7 hours
+MIN_VALIDATOR_WITHDRAWABILITY_DELAY: 256
+# 2**8 (= 256) epochs ~5.7 hours
+SHARD_COMMITTEE_PERIOD: 256
+# 2**10 (= 1024) ~1.4 hour
+ETH1_FOLLOW_DISTANCE: 1024
+
 
+# Validator cycle
+# ---------------------------------------------------------------
+# 2**2 (= 4)
 INACTIVITY_SCORE_BIAS: 4
 # 2**4 (= 16)
 INACTIVITY_SCORE_RECOVERY_RATE: 16
+# 2**4 * 10**9 (= 16,000,000,000) Gwei
+EJECTION_BALANCE: 16000000000
+# 2**2 (= 4)
+MIN_PER_EPOCH_CHURN_LIMIT: 4
+# 2**12 (= 4096)
+CHURN_LIMIT_QUOTIENT: 4096
+# [New in Deneb:EIP7514] 2*
+MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 2
+
+# Fork choice
+# ---------------------------------------------------------------
+# 40%
+PROPOSER_SCORE_BOOST: 40
+# 20%
+REORG_HEAD_WEIGHT_THRESHOLD: 20
```

Comment:
Just curious, we don't define this in the lighthouse ChainSpec. Do we need to?!

Reply:
It's defined on the mainnet one https://github.com/ethereum/consensus-specs/blob/3727a757450feb6110e2d4e403604572a478343d/configs/mainnet.yaml#L98-L100 but probably not used until 
- https://github.com/sigp/lighthouse/pull/4887